index.html

```diff
@@ -1,14 +1,213 @@
+              MORE</a></div>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="footer">
+      <div class="tittle-footer">
+        <h3><span>Questions or Comments?</span><span id="dark"> Get in Touch:</span></h3>
+      </div>
+      <div class="flex-container-footer">
+        <div class="form">
+          <div class="name-email-container">
+            <input 
+            class="name-email" type="text" placeholder="Name">
+            <input class="name-email" type="text" placeholder="Email">
+          </div>
+          <textarea class="message" placeholder="Message"></textarea>
+          <div><a class="cta" href="link to other page"><img class="image-cta" src="./footerlink-email.png" title="CTA">
+              SEND MESSAGE</a></div>
+        </div>
+        <div class="contact-info">
+          <P class="parag">Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime praesentium reprehenderit
+            adipisci id
+            impedit vitae consectetur quas reiciendis.</P>
+          <div class="social">
+            <div class="social-column1">
+              <div class="button-container">
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-home.png" 
+                  title="Address">
+                </div>
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-phone.png"
+                   title="Phone">
+                </div>
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-email.png" 
+                  title="Email">
+                </div>
+              </div>
+              <div class="text-container">
+                <div class="address">
+                  <div class="social-text">1234 Somewhere Road</div>
+                  <div class="social-text">Burlington,VT 00000</div>
+                  <div class="social-text">USA</div>
+                </div>
+                <div class="social-text">(000) 000-000
+                </div>
+                <div class="social-text">info@untitled.tld</div>
+              </div>
+            </div>
+            <div class="social-column2">
+              <div class="button-container">
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-twitter.png" title="Twitter">
+                </div>
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-instagram.png" title="Instagram">
+                </div>
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-dribbble.png" title="Dribbble">
+                </div>
+                <div class="social-buttons">
+                  <img class="image-cta" src="./footerlink-facebook.png" title="Facebook">
+                </div>

```

Comment:
Your footer looks really good overall!  I'm happy to see you using flex-containers so well!  One thing to note, with icons:  avoid using scaling measurements like `vw/vh` because they will look drastically different on different screens.  (Hint: just give them a static height of 20px on `class="image-cta"`, You'll see what I mean!)

Reply:
Also, the 2 columns (`.form` and `.contact-info`) are a bit far apart.  Try centering them (`justify-content`) and giving one of the two a bit of a margin to avoid the text from overlapping.  Should do the trick!